client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -23,13 +34,92 @@ const isMonthsOld = ( months: number, rawDate?: string ) => {
 	return moment().diff( parsedDate, 'months' ) > months;
 };
 
-export default function PlanOnlyThankYou( { primaryPurchase }: PlanOnlyThankYouProps ) {
+export default function PlanOnlyThankYou( {
+	primaryPurchase,
+	isEmailVerified,
+}: PlanOnlyThankYouProps ) {
 	const siteId = useSelector( getSelectedSiteId );
 	const siteSlug = useSelector( getSelectedSiteSlug );
+	const emailAddress = useSelector( getCurrentUserEmail );
 	const siteCreatedTimeStamp = useSelector(
 		( state ) => getSiteOptions( state, siteId ?? 0 )?.created_at
 	);
 
+	const [ resendStatus, setResendStatus ] = useState( RESEND_NOT_SENT );
+
+	const resendEmail = () => {
+		removeNotice( VERIFY_EMAIL_ERROR_NOTICE );
+
+		if ( RESEND_PENDING === resendStatus ) {
+			return;
+		}
+
+		setResendStatus( RESEND_PENDING );
+
+		wpcom.req.post( '/me/send-verification-email', ( error: Error ) => {
+			if ( error ) {
+				errorNotice( translate( "Couldn't resend verification email. Please try again." ), {
+					id: VERIFY_EMAIL_ERROR_NOTICE,
+				} );
+
+				setResendStatus( RESEND_ERROR );
+				return;
+			}
+
+			setResendStatus( RESEND_SUCCESS );
+		} );
+	};
+
+	const resendButtonText = () => {
+		switch ( resendStatus ) {
+			case RESEND_PENDING:
+				return translate( 'Sending…' );
+			case RESEND_SUCCESS:
+				return translate( 'Email sent' );
+			case RESEND_NOT_SENT:
+			case RESEND_ERROR:
+			default:
+				return translate( 'Resend email' );
+		}
+	};
+
+	let subtitle;
+	let headerButtons;
+	if ( primaryPurchase.productSlug === 'ecommerce-bundle' ) {
+		if ( ! isEmailVerified ) {
+			subtitle = translate( "With the plan sorted, it's time to start setting up your store." );
+			headerButtons = (
+				<Button
+					variant="primary"
+					href={ `http://${ siteSlug }/wp-admin/admin.php?page=wc-admin&from-calypso` }
```

Comment:
I feel like there has to be a more elegant way of navigating the user to the desired page, but for a first pass draft PR this felt like an okay starting point. Advice is very welcome on how to clean it up.

Reply:
We can use the `getSiteWooCommerceUrl` function.

```
import getSiteWooCommerceUrl from 'calypso/state/sites/selectors/get-site-woocommerce-url';

...

const siteAdminUrl = useSelector( ( state ) => getSiteWooCommerceUrl( state, siteId ) );
```